autoload/neoformat/formatters/typescript.vim

```diff
@@ -77,3 +77,13 @@ function! neoformat#formatters#typescript#denofmt() abort
         \ 'stdin': 1,
         \ }
 endfunction
+
+function! neoformat#formatters#javascript#biome() abort
```

Comment:
Function name typo here; addressed in https://github.com/sbdchd/neoformat/pull/497